DSharpPlus/Entities/Channel/DiscordChannel.cs

```diff
@@ -574,23 +668,124 @@ public Task<IReadOnlyList<DiscordInvite>> GetInvitesAsync()
         /// <exception cref="Exceptions.NotFoundException">Thrown when the channel does not exist.</exception>
         /// <exception cref="Exceptions.BadRequestException">Thrown when an invalid parameter was provided.</exception>
         /// <exception cref="Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
-        public Task<DiscordInvite> CreateInviteAsync(int max_age = 86400, int max_uses = 0, bool temporary = false, bool unique = false, string reason = null)
-            => this.Discord.ApiClient.CreateChannelInviteAsync(this.Id, max_age, max_uses, temporary, unique, reason);
+        public Task<DiscordInvite> CreateInviteAsync(int max_age = 86400, int max_uses = 0, TargetType target_type = TargetType.Streaming, TargetActivity target_application = TargetActivity.None, bool temporary = false, bool unique = false, string reason = null)
```

Comment:
This is more of a theoretical thing, but maybe the new parameters should go at the end (before `reason`) to reduce backwards compatibility breakage.

Reply:
Agreed, will do

Reply:
Done.